sql/2022/media/image_srcset_sizes_accuracy_pct.sql

```diff
@@ -32,7 +32,7 @@ SELECT
   COUNTIF(image.wDescriptorAbsoluteError > 0) / COUNTIF(image.srcsetHasWDescriptors) AS pct_any_w_descriptor_error,
   COUNTIF(image.actualSizesEstimatedWastedLoadedPixels > 0) / COUNTIF(image.srcsetHasWDescriptors) AS pct_sizes_error_impacted_srcset_selection
 FROM
-  `httparchive.pages.2022_06_01_*` TABLESAMPLE SYSTEM ( 0.1 PERCENT ),
+  `httparchive.pages.2022_06_01_*`,
```

Comment:
@eeeps was this a mistake to leave it in? Do you need to rerun the query with this removed?

Reply:
Looks like this was run without sampling so correct data was saved. Removing this in this PR.